CMakeLists.txt

```diff
@@ -39,6 +39,15 @@ else()
 endif()
 # ------------------
 
+# ------------------
+# Extensive mode for running extension one API tests
```

Comment:
```suggestion
# Extensive mode for running extension oneAPI tests
```
It's "oneAPI" in one word, so please, update it everywhere including name directory.
https://www.oneapi.io/

Reply:
this comment was applied in this commit: ca0b61742388852edfb295a4733924e19be9733c